contrib/mesos/pkg/scheduler/components/framework/framework.go

```diff
@@ -128,14 +123,6 @@ func New(config Config) Framework {
 					return false
 				}
 
-				// the executor IDs must not identify a kubelet-executor with a group that doesn't match ours
```

Comment:
True, we didn't fully solve this problem before. This change, however, results in significantly different cluster behavior. For example, tasks may flap indefinitely because the scheduler repeatedly schedules them on nodes w/ an executor that was created with a different executor-info.

Please add a statement to the  "known issues" doc for this (and perhaps reference the ticket that we've created to track fixing the configuration/command-line problem).